cairo/ethereum/cancun/vm/memory.cairo

```diff
@@ -56,6 +66,12 @@ func memory_write{range_check_ptr, memory: Memory}(start_position: U256, value:
 func memory_read_bytes{memory: Memory}(start_position: U256, size: U256) -> Bytes {
     alloc_locals;
 
+    // Early return if nothing to read
+    if (size.value.low == 0) {
```

Comment:
should this be after the check on `size.value.high = 0` ?
As here in the case size = (0, 1) it should revert instead of returning empty

Reply:
same for buffer_read

Reply:
yes !